test/integration/image-generation/test/index.test.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-env jest */
+import {
+  fetchViaHTTP,
+  findPort,
+  killApp,
+  nextBuild,
+  nextStart,
+} from 'next-test-utils'
+import { join } from 'path'
+
+const appDir = join(__dirname, '../app')
+
+describe('Image Generation', () => {
+  describe('Prod', () => {
+    let app
+    let appPort
+
+    beforeAll(async () => {
+      await nextBuild(appDir)
```

Comment:
Can we ensure the telemetry event is being emitted successfully here similar to https://github.com/vercel/next.js/blob/c962f2d6a4947ff162c3f2c4767e615147e74057/test/integration/telemetry/test/index.test.js#L955

Reply:
Done!